src/main/java/kvp/hhlin/minifullfillment/domain/Sku.java

```diff
@@ -0,0 +1,80 @@
+package kvp.hhlin.minifullfillment.domain;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class Sku {
+
+    private static final int MAX_NAME_LENGTH = 20;
+
+    private String code;
+
+    private String name;
+
+    private String status;
+
+    private Set<Barcode> barcode;
+
+    Sku(String code, String name) {
+        this.code = code;
+        this.name = name;
+        this.status = SkuStatus.valueOf("READY").getStatus();
+        this.barcode = new HashSet<>();
+    }
+
+    private boolean isValidLength(String name) {
+        return name.length() <= MAX_NAME_LENGTH;
+    }
+
+    private boolean isEmpty(String value) {
+        return value.trim().length() == 0 || value.isEmpty() || value.isBlank();
+    }
+
+    public void isValidName(String name) {
+        if (!isValidLength(name) || isEmpty(name)) {
+            throw new IllegalArgumentException("이름은 빈 값일 수 없고 최대 20자입니다.");
+        }
+    }
```

Comment:
우리는 항상 위에서 아래로 읽는게 익숙하기 때문에, 해당 메서드들도 아래와 같이 순서를 바꿔주면 더 잘 읽히지 않을까?

```suggestion
    public void isValidName(String name) {
        if (!isValidLength(name) || isEmpty(name)) {
            throw new IllegalArgumentException("이름은 빈 값일 수 없고 최대 20자입니다.");
        }
    }
    
    private boolean isValidLength(String name) {
        return name.length() <= MAX_NAME_LENGTH;
    }

    private boolean isEmpty(String value) {
        return value.trim().length() == 0 || value.isEmpty() || value.isBlank();
    }
```